DOCS-2035: Adjustments to page manually generating a payment link (#1100)
MultiSafepay/docs: content/tools/multisafepay-control/manually-generated-payment-link.md

read_more: '.'
---

### Manually generated payment link

> For several scenarios, it can be useful for you as a merchant to generate a payment link. The payment link allows the [MultiSafepay Control](https://merchant.multisafepay.com/) to create a unique transaction that the payment can be matched with.

Reviewer: ```suggestion
In several scenarios, it can be useful to generate a payment link. The payment link creates a unique transaction that the payment can be matched with. This transaction can be found in your [MultiSafepay Control](https://merchant.multisafepay.com/)
```
